### PR TITLE
RedisEnterprise - Enable collection of Active/Active metrics

### PR DESCRIPTION
**Pull request by maguec**
### What does this PR do?

Enable collection of Redis Enterprise Active/Active metrics

### Motivation

- Give more visitility in various Redis Enterprise use cases
- Remove a dependency on python requests for the Datadog http wrapper
### Review checklist

- [X] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [X] Feature or bugfix has tests
- [X] Git history is clean
- [X] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Bumped minor version for new features and CHANGELOG is also updated

### CODE REVIEW

**Comment by hithwen** [comment]
Hallo @maguec, thanks for your contribution. It looks like your branch has conflicts with master, can you please address those?

**Comment by maguec** [comment]
@hithwen  resoved the conflict

File: redisenterprise/README.md [hunk 1]
```diff
@@ -112,7 +115,7 @@ All [Redis Enterprise events][10] are collected.
 
 ## Troubleshooting
 
-Contact the [Redis Enterprise Support Team][11]
+Contact the [Redis Field Engineering Team][11]
```
Comment (ruthnaebeck) [inline redisenterprise/README.md:118]:
```suggestion
Contact the [Redis Field Engineering Team][11].
```

File: redisenterprise/metadata.csv [hunk 2]
```diff
@@ -47,3 +47,10 @@ redisenterprise.total_node_cores,gauge,,item,,Number of CPU cores in the Redis E
 redisenterprise.total_node_memory,gauge,,byte,,Byte of memory in the Redis Enterprise Cluster,0,redis_enterprise,Redis Enterprise Cluster Memory Available
 redisenterprise.total_node_count,gauge,,item,,Number of nodes in the Redis Enterprise Cluster,0,redis_enterprise,Redis Enterprise Cluster Node count
 redisenterprise.total_active_nodes,gauge,,item,,Number of active nodes in the Redis Enterprise Cluster,0,redis_enterprise,Redis Enterprise Cluster Active Node count
+redisenterprise.crdt_egress_bytes,gauge,,byte,,Bytes being sent to CRDT peer,0,redis_enterprise,Redis Enterprise Database Peer Bytes Sent
```
Comment (ruthnaebeck) [inline redisenterprise/metadata.csv:50]:
```suggestion
redisenterprise.crdt_egress_bytes,gauge,,byte,,The bytes being sent to the CRDT peer,0,redis_enterprise,Redis Enterprise Database Peer Bytes Sent
```

File: redisenterprise/metadata.csv [hunk 3]
```diff
@@ -47,3 +47,10 @@ redisenterprise.total_node_cores,gauge,,item,,Number of CPU cores in the Redis E
...
+redisenterprise.crdt_egress_bytes_decompressed,gauge,,byte,,Decompressed bytes being sent to CRDT peer,0,redis_enterprise,Redis Enterprise Database Peer Bytes Sent Decompressed
```
Comment (ruthnaebeck) [inline redisenterprise/metadata.csv:51]:
```suggestion
redisenterprise.crdt_egress_bytes_decompressed,gauge,,byte,,The decompressed bytes being sent to the CRDT peer,0,redis_enterprise,Redis Enterprise Database Peer Bytes Sent Decompressed
```

File: redisenterprise/metadata.csv [hunk 4]
```diff
@@ -47,3 +47,10 @@ redisenterprise.total_node_cores,gauge,,item,,Number of CPU cores in the Redis E
...
+redisenterprise.crdt_ingress_bytes,gauge,,byte,,Bytes being received from CRDT peer,0,redis_enterprise,Redis Enterprise Database Peer Bytes Received
```
Comment (ruthnaebeck) [inline redisenterprise/metadata.csv:52]:
```suggestion
redisenterprise.crdt_ingress_bytes,gauge,,byte,,The bytes being received from the CRDT peer,0,redis_enterprise,Redis Enterprise Database Peer Bytes Received
```

File: redisenterprise/metadata.csv [hunk 5]
```diff
@@ -47,3 +47,10 @@ redisenterprise.total_node_cores,gauge,,item,,Number of CPU cores in the Redis E
...
+redisenterprise.crdt_ingress_bytes_decompressed,gauge,,byte,,Decompressed bytes being received from CRDT peer,0,redis_enterprise,Redis Enterprise Database Peer Bytes Received Decompressed
```
Comment (ruthnaebeck) [inline redisenterprise/metadata.csv:53]:
```suggestion
redisenterprise.crdt_ingress_bytes_decompressed,gauge,,byte,,The decompressed bytes being received from the CRDT peer,0,redis_enterprise,Redis Enterprise Database Peer Bytes Received Decompressed
```

File: redisenterprise/metadata.csv [hunk 6]
```diff
@@ -47,3 +47,10 @@ redisenterprise.total_node_cores,gauge,,item,,Number of CPU cores in the Redis E
...
+redisenterprise.crdt_local_lag,gauge,,second,,Local lag in CRDT applies,0,redis_enterprise,Redis Enterprise CRDT lag
```
Comment (ruthnaebeck) [inline redisenterprise/metadata.csv:54]:
```suggestion
redisenterprise.crdt_local_lag,gauge,,second,,The local lag in the CRDT applies,0,redis_enterprise,Redis Enterprise CRDT lag
```

File: redisenterprise/metadata.csv [hunk 7]
```diff
@@ -47,3 +47,10 @@ redisenterprise.total_node_cores,gauge,,item,,Number of CPU cores in the Redis E
...
+redisenterprise.crdt_pending_max,gauge,,item,,Local pending writes in CRDT max,0,redis_enterprise,Redis Enterprise CRDT Pending Max
```
Comment (ruthnaebeck) [inline redisenterprise/metadata.csv:55]:
```suggestion
redisenterprise.crdt_pending_max,gauge,,item,,The local pending writes in the CRDT max,0,redis_enterprise,Redis Enterprise CRDT Pending Max
```

File: redisenterprise/metadata.csv [hunk 8]
```diff
@@ -47,3 +47,10 @@ redisenterprise.total_node_cores,gauge,,item,,Number of CPU cores in the Redis E
...
+redisenterprise.crdt_pending_min,gauge,,item,,Local pending writes in CRDT min,0,redis_enterprise,Redis Enterprise CRDT Pending Min
```
Comment (ruthnaebeck) [inline redisenterprise/metadata.csv:56]:
```suggestion
redisenterprise.crdt_pending_min,gauge,,item,,The local pending writes in the CRDT min,0,redis_enterprise,Redis Enterprise CRDT Pending Min
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]
Docs review

**Review by fanny-jiang — APPROVED** [review]

**Comment by fanny-jiang** [comment]
@ruthnaebeck 👍🏼  on the docs side?

**Review by apigirl — APPROVED** [review]

**Comment by yzhan289** [comment]
Thanks for the PR. I just merged this in and am working on getting it updated on Datadog.

**Comment by yzhan289** [comment]
Hi @maguec, the dashboard is currently not able to show up on Datadog since the file was not added to `manifest.json`. I have made a PR adding it: https://github.com/DataDog/integrations-extras/pull/1076

**Pull request merged**